l10n_ec_ote/security/ir.model.access.csv

```diff
@@ -0,0 +1,2 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_l10n_ec_parish,l10n.ec.parish,model_l10n_ec_parish,sales_team.group_sale_salesman,1,0,0,0
```

Comment:
Please add  group  `base.group_partner_manager` to `CRUD`
![image](https://user-images.githubusercontent.com/7775116/226715017-5160172f-a8eb-4fce-925d-52cc2cbbe2a3.png)